torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -1690,12 +1677,23 @@ c10::List<Elem> makeListForDictValues(
     const std::vector<std::pair<IValue, IValue>>& order) {
   c10::List<Elem> values;
   values.reserve(order.size());
-  for (auto item : order) {
+  for (const auto& item : order) {
     values.push_back(item.second.to<Elem>());
   }
   return values;
 }
 
+template <>
+c10::impl::GenericList makeListForDictValues<IValue>(
```

Comment:
where do we use the generic form of this?

Reply:
https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/register_prim_ops.cpp#L1736